unit-test-coverage.yaml

```diff
@@ -3,7 +3,7 @@ packages:
   cmd/modulectl/create: 100
   internal/common/validation: 92
   internal/service/scaffold: 91
-  internal/service/contentprovider: 87.4
```

Comment:
Changes look good so far. But can you figure out what tests are missing here to tick the 100 ?
`SecurityConfig`'s functionality kind of small so it should be easily possible.